src/plugins/intel_gpu/include/intel_gpu/runtime/debug_configuration.hpp

```diff
@@ -41,6 +41,7 @@ enum class LogLevel : int8_t {
 }  // namespace ov
 
 #ifdef GPU_DEBUG_CONFIG
+#define __FILENAME__ (strrchr(__FILE__, '/') ? strrchr(__FILE__, '/') + 1 : __FILE__)
```

Comment:
does it work for windows as well?

Reply:
Windows has an issue for different separator issue, I updated to use different separator in Windows.